res/css/components/views/beacon/_BeaconStatus.pcss

```diff
@@ -67,7 +67,7 @@ limitations under the License.
 }
 
 .mx_BeaconStatus_label {
-    margin-bottom: 2px;
+    margin-bottom: $spacing-2;
```

Comment:
I thought the whole point of the $spacing variables was to encourage people not to use values which are not a multiple of 4?